### ISSUE TITLE
Button accessibility

### ISSUE THREAD

**Issue by bluefantail** [powershop/fluxfederation.com#15]
Currently `flux-button` fails an accessibility check - we might want to consider given this is used on some of the key calls to action.

![image (9)](https://user-images.githubusercontent.com/616368/55852872-ca6da380-5bb2-11e9-9c18-eb458a4a26d7.png)

**Comment by bluefantail** [comment]
Passes Aa accessibility standard now - fixed in 211b7b6.
